server/src/js/components/map/layers.js

```diff
@@ -39,18 +39,54 @@ window.flood.maps.layers = {
   // Tile layers
   //
 
+  topography: () => {
+    return new TileLayer({
+      ref: 'road',
+      source: new BingMaps({
+        key: window.flood.model.bingMaps + '&c4w=1&cstl=rd&src=h&st=me|lv:0_trs|v:0_pt|v:0',
+        imagerySet: 'RoadOnDemand'
+      }),
+      visible: false,
+      zIndex: 0
+    })
+  },
+
   road: () => {
     return new TileLayer({
       ref: 'road',
       source: new BingMaps({
-        key: window.flood.model.bingMaps, // + '&c4w=1&cstl=rd&src=h&st=ar|fc:b5db81_wt|fc:a3ccff_tr|fc:50a964f4;sc:50a964f4_ard|fc:ffffff;sc:ffffff_rd|fc:50fed89d;sc:50eab671;lbc:626a6e_mr|lbc:626a6e_hr|lbc:626a6e_st|fc:ffffff;sc:ffffff_g|lc:dfdfdf_trs|lbc:626a6e',
+        key: window.flood.model.bingMaps,
         imagerySet: 'RoadOnDemand'
       }),
       visible: false,
       zIndex: 0
     })
   },
 
+  // road: () => {
```

Comment:
Does this need leaving in?

Reply:
no, was left in for reference but doesnt need to be there, ill remove it